ciao-launcher/tests/ciao-launcher-server/server.go

```diff
@@ -491,6 +495,8 @@ func createConfigFile(confPath string) error {
 	conf.Configure.Launcher.MemoryLimit = memLimit
 	conf.Configure.Launcher.ComputeNetwork = []string{computeNet}
 	conf.Configure.Launcher.ManagementNetwork = []string{mgmtNet}
+	conf.Configure.Storage.SecretPath = secretPath
```

Comment:
So for the server I was thinking of making these required to be passed in but again looking for feedback on use of default values.